apps/agent/scripts/buidler-hooks.js

```diff
@@ -0,0 +1,68 @@
+let voting, accounts
+
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+module.exports = {
+  postDao: async function ({ _experimentalAppInstaller, log }, bre) {
+    const bigExp = (x, y) =>
+      bre.web3.utils
+        .toBN(x)
+        .mul(bre.web3.utils.toBN(10).pow(web3.utils.toBN(y)))
+    const pct16 = (x) => bigExp(x, 16)
+
+    // Retrieve accounts.
+    accounts = await bre.web3.eth.getAccounts()
+
+    // Deploy a minime token an generate tokens to root account
+    const minime = await _deployMinimeToken(bre)
+    await minime.generateTokens(accounts[1], pct16(10))
+    log(`> Minime token deployed: ${minime.address}`)
+
+    const tokens = await _experimentalAppInstaller('token-manager', {
+      skipInitialize: true,
+    })
+
+    await minime.changeController(tokens.address)
+    log(`> Change minime controller to tokens app`)
+    await tokens.initialize([minime.address, true, 0])
+    log(`> Tokens app installed: ${tokens.address}`)
+
+    voting = await _experimentalAppInstaller('voting', {
+      initializeArgs: [
+        tokens.address,
+        pct16(50), // support 50%
+        pct16(25), // quorum 15%
+        604800, // 7 days
+      ],
+    })
+    log(`> Voting app installed: ${voting.address}`)
+
+    await tokens.createPermission('MINT_ROLE', voting.address)
+    await voting.createPermission('CREATE_VOTES_ROLE', tokens.address)
+  },
+  postInit: async function ({ proxy, _experimentalAppInstaller, log }, bre) {
```

Comment:
I realize in cases like Agent we most likely want to have a way to assign custom permissions to the proxy. I'm adding an issue to track this addition.

Reply:
Just wondering, why do you think it's important to install the other apps here?

What if we just installed the Agent app during local development (maybe exposing a flag or second "development" script later on if we wanted a more complete set up)?

Reply:
I thought it could be interesting to have a setup to test Agent interactions. But let's keep it simple for now and we can do specific examples on a dedicated repo maybe.